packages/nimble-components/src/anchor/index.ts

```diff
@@ -1,4 +1,4 @@
-import { attr } from '@microsoft/fast-element';
+import { attr, observable } from '@microsoft/fast-element';
```

Comment:
I noticed an unexpected behavior while testing:
1. tab focus to the table
2. down arrow a few times to a row
3. tab to give focus to an action menu in that row
4. enter to open the action menu, arrow to an item, enter to select it
5. shift tab to navigate between action menus

Unexpected behavior: it requires two shift-tabs to move focus from the first column's action menu to outside the table. If you skip step 4 it behaves as expected, where the first shift-tab moves focus from the first action menu to outside the table. Once it enters the bad state it seems to stay in that state until a page refresh.

Tested in macOS Firefox.

Reply:
I can't reproduce this on Firefox in Windows, so you might have to demo it to me.

Can you tell where tab focus is after the first Shift-Tab, before you have to do it a 2nd time?

Reply:
Tab focus remains on the action menu button after the first Shift-Tab.

I tried in macOS Edge and didn't see the same bug.

I tried in macOS Safari and saw a different bug: regardless of whether you open any of the action menus, Shift-tab from the first action menu takes you to the focusable content AFTER the table rather than before. Similarly, tab from the last action menu takes you to the focusable content BEFORE the table rather than after. 

Happy to jump on a call and debug together, just ping me when you're ready to look at it.

Reply:
Gave Jesse a new build to try and I think I have a fix for this behavior (will be in a future update).

Reply:
Pushed updates that should fix this.